### PR TITLE
Redundancy: Add `remove_non_standard_import` taste

### PR DESCRIPTION
**Pull request by adhikasp**
Relates to https://github.com/coala/coala-bears/issues/2016

### CODE REVIEW

File: coalib/bearlib/aspects/Redundancy.py [hunk 1]
```diff
@@ -97,6 +97,16 @@ class docs:
         Usually, unused imports can simply be removed.
         """
 
+    remove_non_standard_import = Taste[bool](
+        "Remove ALL unused import, include those not from language's "
+        'standard library.',
+        (True, False), default=True)
+
+    expand_wildcard_imports = Taste[bool](
```
Comment (jayvdb) [inline coalib/bearlib/aspects/Redundancy.py]: A star import that can be expanded is by very definition  'used' , and doesnt belong in this aspect.

**Comment by jayvdb** [comment]
Also add tests.

**Comment by adhikasp** [comment]
> Also add tests.

Um this is aspect definition, what do you mean by test?

File: coalib/bearlib/aspects/Redundancy.py (hunk 1 above)
Comment (adhikasp) [inline coalib/bearlib/aspects/Redundancy.py]: Note that this feature still hasn't released from upstream so I think I'll just remove this.

**Comment by jayvdb** [comment]
A constant variable is created; verify the object in the created variable is what you want it to be.

**Comment by jayvdb** [comment]
ack 20c316f

**Comment by Nosferatul** [comment]
rebuilded

**Comment by yukiisbored** [comment]
reack https://github.com/coala/coala/pull/4659/commits/4dff6f5ccfa6177a0b91533492462f0dc9a132a2

**Comment by yukiisbored** [comment]
@rultor merge

**Pull request merged**